src/main/cpp/utils/Logger.h

```diff
@@ -100,72 +115,66 @@ class Logger
-            const std::string&      message                 
+            const std::string&      locationIdentifier,
+            const std::string&      message
         );
 
-
-        /// @brief Write a message to the dashboard
-        /// @param [in] std::string: classname or object identifier
-        /// @param [in] std::string: message
-        void OnDash
-        (
-            const std::string&   locationIdentifier,    
-            const std::string&   message  
-        );              
-
-        /// @brief Write a message to the dashboard
+        /// @brief Write a message and value to the dashboard, for monitoring the value
         /// @param [in] std::string: classname or object identifier
-        /// @param [in] bool: boolean that should be written
+        /// @param [in] double: value that should be written (int or bool could also be 'doubled' and sent)
         void OnDash
         (
-            const std::string&   locationIdentifier,     
-            bool                 value                 
+            const std::string&   locationIdentifier,
+            double               value
         );
 
 
+        //
+        //  ToNtTable() sends an identifier and information (string or value) to the specified network table.
+        //  Anything on the network can then read the message.
+        //
         void ToNtTable
         (
             const std::string&  ntName,
             const std::string&  identifier,
-            const std::string&  string 
+            const std::string&  string
         );
 
         void ToNtTable
         (
             const std::string&  ntName,
             const std::string&  identifier,
-            double              value 
+            double              value
         );
 
         void ToNtTable
         (
             std::shared_ptr<nt::NetworkTable>   ntable,
             const std::string&                  identifier,
-            const std::string&                  msg 
+            const std::string&                  msg
         );
+
         void ToNtTable
         (
             std::shared_ptr<nt::NetworkTable>   ntable,
             const std::string&                  identifier,
-            double                              value 
+            double                              value
         );
 
 
-
     protected:
 
 
     private:
         Logger();
         ~Logger() = default;
 
-        LOGGER_OPTION           m_option;
-        LOGGER_LEVEL            m_level;
+        LOGGER_OPTION           m_option;               // indicates where the message should go
+        LOGGER_LEVEL            m_level;                // the level at which a message is important enough to send
         std::set<std::string>   m_alreadyDisplayed;
         static Logger*          m_instance;
+        int                     m_CyclingCounter;       // count 20ms loops
 
-
+        frc::SendableChooser<LOGGER_OPTION>     m_OptionChooser;

```

Comment:
should be m_optionChooser and m_levelChooser

How are these used, I see them in the constructor, but nowhere elese